openapi/components/headers/X-RateLimit-Limit.yaml

```
@@ -0,0 +1,4 @@
+description: Total number of rate limit tokens for this request.
```

suggestion: to add more detail

I think it would be good to point to the Rate limit section to provide more information 

```suggestion
description: |-
  Total number of rate limit tokens for this request.
  For more information, see [Rate limits](#section/Rate-limits).
```
It may also be good to add `rate limit period` to the description 
```suggestion
description: |-
  Total number of rate limit tokens for this request within a rate limit period.
  For more information, see [Rate limits](#section/Rate-limits).
```